fuel/app/tests/helper/common.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * @group Helper
+ */
+class Test_Helper_Common extends TestCase
+{
+	/**
+	 * @before
+	 */
+	protected function truncate_data()
+	{
+		DBUtil::truncate_table('messages');
+	}
+
+	public function test_format_time()
+	{
+		$expect = "10/05/2015 12:10";
+		$date   = new DateTime($expect);
+		$date   = $date->getTimestamp();
+
+		$this->assertEquals(Helper_Common::format_time($date), $expect);
+	}
+
+	/**
+	 * @dataProvider data_insert_data
+	 */
+	public function test_insert_data($input,$expect)
+	{
+		Helper_Common::install_data($input['data'],$input['table']);
+		$result = Model_Message::get_all();
```

Comment:
@luong-phuc-mulodo i think its better to truncate table at the start of this test because assertion will be failed when some data existed in the table already.